doc/dev.rst

```diff
@@ -249,49 +260,139 @@ What kind of issue is it?
   a new feature proposal or refactoring. Should have steps to reproduce with
   desired and actual behavior.
 
-:code:`help wanted`
-  The core team does not plan to address this issue, perhaps because we don't
-  know how, but we think it would be good to address it. We hope someone from
-  the community will volunteer.
+:code:`refactor`
+  Change that will improve Charliecloud but does not materially affect
+  user-visible behavior.
+
+How important/urgent is it?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-:code:`key issue`
-  A particularly important or notable issue.
+Choose *one priority* from:
 
-:code:`question`
-  Support request that does not report a problem or ask for a change. Close
-  these after the question is answered or several days with no activity.
+:code:`high`
+  High priority. If an issue is blocking someone's work it very likely belongs
+  here.
 
-What do we plan to do about it?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+:code:`medium`
+  Medium priority.
 
-For all of these, leave other tags in place, e.g. :code:`bug`.
+:code:`low`
+  Low priority. Note: Unfortunately, due to resource limitations, complex
+  issues here are likely to wait a long time, perhaps forever. If that makes
+  you particularly sad on a particular issue, please comment to say why. Maybe
+  it's mis-prioritized.
 
 :code:`deferred`
   No plans to do this, but not rejected. These issues stay open, because we do
   not consider the deferred state resolved. Submitting PRs on these issues is
   risky; you probably want to argue successfully that it should be done before
   starting work on it.
 
-:code:`duplicate`
-  Same as some other previously reported issue. In addition to this tag,
-  duplicates should refer to the other issue and be closed.
+Note that (1) the priority levels are indeed fuzzy, and we have no plans to
+make them more precise; (2) issues are not a priority queue, partly for this
+reason, i.e., we do work on lower-priority issues while higher-priority ones
+are still open; and (3) issues often move between priority levels. In
+particular, if you think we picked the wrong priority level, please say so.
+
+What part of Charliecloud is affected?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Choose *one or more components* from:
+
+:code:`runtime`
+  The container runtime itself; largely :code:`ch-run`.
+
+:code:`image`
+  Image building and interaction with image registries; largely
+  :code:`ch-image`. (Not to be confused with image management tasks done by
+  glue code.)
+
+:code:`glue`
```

Comment:
I don't think `glue` is descriptive or informative. Unlike `image`, `runtime`, `build`, and `doc`, it doesn't immediately correspond to some action or idea. What about `script`?

Reply:
Well `bin/ch-image` is unfortunately a script too. I agree `glue` isn't the best.

Reply:
Though `glue` has an emoji (🍝) while `script` does not.

Reply:
> Well `bin/ch-image` is unfortunately a script too. I agree `glue` isn't the best.

Hmm, since we are typically referring the bash/sh scripts when we refer to glue, can we do something like `POSIX sh` and/or `bash`?

Reply:
In the interest of moving forward, I'm going to call it `glue` for now. But we can easily change it later.